docs/ref/checks.txt

```diff
@@ -868,6 +868,8 @@ The following checks are performed on the default
   for its builtin permission names to be at most 100 characters.
 * **auth.E012**: The permission codenamed ``<codename>`` of model ``<model>``
   is longer than 100 characters.
+* **auth.E013**: The login required middleware requires authentication
+  middleware to be installed, and come before the login required middleware.
```

Comment:
This feels a tiny odd, what about:
```suggestion
  middleware to be installed, and should be defined before the login required middleware.
```

Reply:
Just so you know where this is coming from, this is model-ed off https://github.com/django/django/blob/f030236a86a64a4befd3cc8093e2bbeceef52a31/django/contrib/messages/storage/session.py#L17

Shall I still update the wording or keep it to be the same format as this other error

Reply:
My proposal would be to update the error message in this PR (both in the exception raising code and in the docs), since it reads awkwardly, and follow up with a separated PR to fix phrasing for the session middleware. What do you think?

Reply:
Error message is updated 👍